test/sysv_sliding_window_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class TestSysVSlidingWindow < MiniTest::Unit::TestCase
+  CLASS = ::Semian::SysV::SlidingWindow
+
+  def setup
+    @sliding_window = CLASS.new(max_size: 6,
+                                name: 'TestSysVSlidingWindow',
+                                permissions: 0660)
+    @sliding_window.clear
+  end
+
+  def teardown
+    @sliding_window.destroy
+  end
+
+  include TestSimpleSlidingWindow::SlidingWindowTestCases
```

Comment:
No, you make a good point, let's wait with them—but make sure to not forget them. We can't merge this unless it's green.